services/bstats/bstats.tester.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
```

Comment:
We should have a `tester` for each defined `service` so if you're going to have different services for `servers` and `players` then let's split this into two different tester files (`bstats-servers.tester.js` and `bstats-players.tester.js`)